src/components/nowPlayingBar/nowPlayingBar.js

```diff
@@ -822,3 +850,5 @@ document.addEventListener('viewbeforeshow', function (e) {
         }
     }
 });
+
+document.addEventListener('keydown', handleVolumeKeyInput);
```

Comment:
Event listeners should be added/removed in the same place as all the others.

Reply:
@thornbill I don't understand your ask here.  

The `addEventListener` is literally on the next line after the existing event listener for this component.  See line with event listener already in this file: https://github.com/jellyfin/jellyfin-web/pull/5614/files/6b568edca381c5d4ddc6cc59df1a7fcd80161482#diff-d43a0a5c9a2354bd74a607e4a1c94f73a2415d3b7ec6b3802a34f6e14926b0c0R836

If there is another location, eg. "the same place as all the others" - it is unclear which file your are referring to.